c2rust-analyze/tests/filecheck.rs

```diff
@@ -60,7 +60,13 @@ fn filecheck() {
     let filecheck_bin = env::var_os("FILECHECK")
         .map(PathBuf::from)
         .or_else(|| detect_filecheck().map(|it| it.to_owned()))
```

Comment:
`$(llvm-config --bindir)/FileCheck` should replace `detect_filecheck`, so we no longer need to use or have `detect_filecheck`.

Reply:
fixed in [fb5cc98](https://github.com/immunant/c2rust/pull/767/commits/fb5cc98850db169a791ab7cd43d0aa66f42af379) and  [6b6c038](https://github.com/immunant/c2rust/pull/767/commits/6b6c03833edb9c177e83eed836265fa2a4376d52)